readme.md

```diff
@@ -1,15 +1,17 @@
-# Super Secret Puzzle Tools to Win
+# Super Secret Puzzle Tools to Win [![Build Status](https://travis-ci.com/s-zhang/puzzlehunt-tools.svg?branch=master)](https://travis-ci.com/s-zhang/puzzlehunt-tools)
 
-## Tools to Create
+## puzzle-utils
+
+## Features/To-Dos
 (Sign up for whatever you decide to work on here so we don't duplicate work)
 
 Basically what I want is all the random online tools in one easy lightweight place or in script version.
 
-- Word Ladder tools
+- Word Ladder tools (TODO)
```

Comment:
Wasn't the plan to open issues on these?

Reply:
I realized it'd be nice for people to see what are the features that are and will be in this toolset. We can still create issues to track progress, thoughts on these items.